packages/runtime/test/consumption/iqlQuery.test.ts

```diff
@@ -45,17 +46,19 @@ describe("IQL Query", () => {
             }
         });
 
-        rLocalAttribute = response.value;
+        rLocalAttribute1 = response.value;
```

Comment:
I think you can also write the value of the variable `response` directly here instead of defining an auxiliary variable if you do the same for `rLocalAttribute2`.